src/cc/tests/common/StrUtils_T.cc

```diff
@@ -0,0 +1,28 @@
+#include <gtest/gtest.h>
+
+#include "common/StrUtils.h"
+
+namespace KFS {
+namespace Test {
+
+TEST(StrUtils_T, printf) {
```

Comment:
These aren't fixture tests unfortunately. I don't want to replace the suggestion in the Developer Documentation with this just yet. However, I have added a mention of checking out some of the existing tests for some examples.